src/api/test-helpers/create-full-application.ts

```diff
@@ -180,6 +190,30 @@ export const createFullApplication = async (context: Context, policyType?: strin
     query: 'id hasAntiBriberyCodeOfConduct',
   })) as ApplicationDeclaration;
 
+  // gets financialUk id from application for updating
+  const updatedApplication = (await context.query.Application.findOne({
+    where: { id: application.id },
+    query: 'id nominatedLossPayee { id isAppointed financialUk { id } }',
+  })) as Application;
+
+  // updates nominatedLossPayee
+  (await context.query.NominatedLossPayee.updateOne({
+    where: {
+      id: updatedApplication.nominatedLossPayee.id,
+    },
+    data: mockNominatedLossPayee,
+    query: 'id',
+  })) as ApplicationNominatedLossPayee;
+
+  // updates financialUk table
+  (await context.query.LossPayeeFinancialUk.updateOne({
+    where: {
+      id: updatedApplication.nominatedLossPayee.financialUk.id,
+    },
+    data: mockLossPayeeFinancialDetailsUk,
+    query: 'id accountNumber sortCode',
```

Comment:
Do we need `accountNumber` and `sortCode` in the query? Maybe just `id` is enough